cpp/openlocationcode.cc

```diff
@@ -55,11 +54,15 @@ double compute_precision_for_length(int code_length) {
   return pow_neg(internal::kEncodingBase, -3) / pow(5, code_length - 10);
 }
 
-// Finds the position of a char in the encoding alphabet.
+// Returns the position of a char in the encoding alphabet, or -1 if invalid.
 int get_alphabet_position(char c) {
-  const char* end = internal::kAlphabet + internal::kEncodingBase;
-  const char* match = std::find(internal::kAlphabet, end, c);
-  return (end == match)? -1 : (match - internal::kAlphabet);
+  // Lookup table of alphabet positions of characters 'C' through 'X'.
```

Comment:
Can you add more context to the comment? This is harder to follow than using std::find, so I want to make sure we understand what's going on when we look back at it in a couple months. Something like the below:

// Lookup table of the alphabet positions of characters 'C' through 'X', inclusive. A value of -1 means the character isn't part of the alphabet. We use a lookup table for performance reasons (e.g. over std::find).

Reply:
Done.